packages/node-resolve/src/index.js

```diff
@@ -258,9 +258,20 @@ export function nodeResolve(opts = {}) {
       idToPackageInfo.set(resolved, packageInfo);
 
       if (hasPackageEntry) {
-        if (builtins.has(resolved) && preferBuiltins && isPreferBuiltinsSet) {
```

Comment:
Pretty sure `resolved === importee` here because we were depending on `resolve()` agreeing on it being a builtin and resolving to the same thing (given we didn't add ```importSpecifierList.push(`${importee}`)```). Therefore `builtins.has(resolved)` is the same as `importeeIsBuiltin`.

Reply:
^ @lukastaegert

Reply:
👍